.docs/content/1.installation/3.aws/2. aws-deployment-troubleshooting.md

```diff
@@ -0,0 +1,115 @@
+# AWS Deployment Troubleshooting Guide 🛠️
+
+This guide covers common issues you might encounter while deploying your application on AWS and provides solutions to address them.
+
+## 1. Error: Partition Not Found
+
+If you encounter a Partition Not Found error during deployment or runtime, ensure the following:
+
+- Set the Replication Factor to 1 in the **parameters.tfvars**:
```

Comment:
reference the file
```suggestion
- Set the Replication Factor to 1 in the [**parameters.tfvars**](infrastructure/quick-deploy/aws/parameters.tfvars):
```
Not sure it will work like that, you may have to remove the **